srcpkgs/python3-jedi/template

```diff
@@ -11,10 +11,10 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://jedi.readthedocs.io/"
 distfiles="${PYPI_SITE}/j/jedi/jedi-${version}.tar.gz"
-checksum=bae794c30d07f6d910d32a7048af09b5a39ed740918da923c6b780790ebac612
+checksum=bcf9894f1753969cbac8022a8c2eaee06bfa3724e4192470aaffe7eb6272b0c4
 
 do_check() {
-	PYTHONPATH="$(cd build/lib* && pwd)" python3 ./test/run.py
+	python3 -m pytest
```

Comment:
this is the default do_check for this build_style, more-or-less

Reply:
Not sure why but it causes test failures for jedi: https://github.com/void-linux/void-packages/actions/runs/6376660366/job/17304005993?pr=46180#step:7:2386

Tests pass with this change but I don't know if it is because that case is being skipped or not: https://github.com/void-linux/void-packages/actions/runs/6364708653/job/17281167107#step:7:2391

Reply:
what I'm saying is `python3 -m pytest` is the same as what will happen if `do_check() { ... }` is removed, so you can remove it

Reply:
Thanks a lot. I have removed that bit from the template.